transformer/kubernetes/apiresource/service.go

```diff
@@ -360,21 +360,21 @@ func (d *Service) createIngress(ir irtypes.EnhancedIR, targetCluster collecttype
 	if len(hostHTTPIngressPaths) == 0 {
 		return nil
 	}
+	qaLabel := targetCluster.Labels[collecttypes.ClusterQaLabelKey]
```

Comment:
will this always succeed? can `qaLabel` be empty string?

Reply:
addressed.